<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NODE1129" ignored="0" name="Test" tests="3" failures="2" timestamp="10 Jul 2018 07:56:18 GMT" time="0.043" errors="0">
  <testcase name="verifyConditionOne" time="0.011" classname="Jen.JenkinsBuild.AppTest"/>
  <testcase name="verifyConditionTwo" time="0.0" classname="Jen.JenkinsBuild.AppTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Jen.JenkinsBuild.AppTest.verifyConditionTwo(AppTest.java:30)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyConditionTwo -->
  <testcase name="verifyConditionThree" time="0.0" classname="Jen.JenkinsBuild.AppTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Jen.JenkinsBuild.AppTest.verifyConditionThree(AppTest.java:43)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyConditionThree -->
</testsuite> <!-- Test -->
